libecho/tests/tmp_test.cpp

```diff
@@ -4,15 +4,15 @@
 #include <QtCore/QFile>
 
 TEST(test_send, tmp_test) {
-    /*QFile sourceFile;
+    QFile sourceFile;
     sourceFile.setFileName("/dev/urandom");
     sourceFile.open(QIODevice::ReadOnly);
-    auto urandom = sourceFile.read(1000);
+    auto urandom = sourceFile.read(10000);
```

Comment:
warto to chyba gdzieś wrzucić jako stałą?

Reply:
Moge to zmienić, ale zarówno ciała funkcji w echo.cc jak i kod testów to jedynie tymczasowy kod tylko po to żebym widział czy wogóle coś działa, i będzie za chwilę zaorany